### PR TITLE
Add info logging output.

### PR DESCRIPTION
**Pull request by rbartlensky**
Users can now trace the execution of the pwvlt crate with `-v`.

### CODE REVIEW

File: pwvlt-cli/src/config.rs [hunk 1]
```diff
@@ -23,20 +23,28 @@ pub fn load_config() -> Result<Config, Error> {
     // config will be loaded from ~/.config/pwvlt/config.toml
     let config_folder = home.join(".config").join("pwvlt");
     if !config_folder.exists() {
+        log::info!("Creating {}", config_folder.display());
```
Comment (gabi-250) [inline pwvlt-cli/src/config.rs]: Minor, but s/folder/dir?

File: pwvlt-cli/src/config.rs [hunk 2]
```diff
@@ -23,20 +23,28 @@ pub fn load_config() -> Result<Config, Error> {
...
         create_dir_all(&config_folder)?;
     }
     let config_file = config_folder.join("config.toml");
     if !config_file.exists() {
-        write!(File::create(&config_file)?, "{}", DEFAULT_CONFIG)?;
+        log::info!("Creating {}", config_file.display());
```
Comment (gabi-250) [inline pwvlt-cli/src/config.rs]: I think this is redundant given that we are logging this below as well.

File: pwvlt-cli/src/main.rs [hunk 3]
```diff
@@ -105,10 +136,13 @@ fn main() {
     if let Err(e) = handle_args(matches) {
         match e {
             Error::HomeNotFound => error!("Couldn't find home directory."),
-            Error::Io(e) => error!("An IO error occurred: {}", e),
+            Error::Io(e) => error!(
+                "An IO error occurred during parsing the configuration: {}",
```
Comment (gabi-250) [inline pwvlt-cli/src/main.rs]: s/during/while

**Comment by rbartlensky** [comment]
@gabi-250 Done

**Comment by gabi-250** [comment]
Looks good! You can squash if you want.

**Comment by rbartlensky** [comment]
@gabi-250 done

**Pull request merged**